templates/tonic-web-server-ingress.yaml

```diff
@@ -14,6 +18,13 @@ metadata:
     nginx.ingress.kubernetes.io/proxy-read-timeout: "3600" # 1 hour
     nginx.ingress.kubernetes.io/proxy-send-timeout: "3600" # 1 hour
     nginx.ingress.kubernetes.io/proxy-body-size: "0" # disable body size constraint
+  {{- end }}
+  {{- if .Values.tonicai.ingress.labels }}
+  labels:
+    {{- with .Values.tonicai.ingress.labels }}
+    {{ toYaml . | nindent 4 }}
+    {{- end }}
+  {{- end }}
 spec:
```

Comment:
```suggestion
spec:
  {{- if .Values.tonicai.ingress.class }}
  ingressClassName: {{ .Values.tonicai.ingress.class }}
  {{- end }}
```

Reply:
Looks good other wise, I'll merge after this change.

Reply:
@anr-tonic This has been added.